eds/scripts/scripts.js

```diff
@@ -243,6 +243,27 @@ async function loadPage() {
   loadDelayed();
 }
 
+/**
+* convert link button to calcite button
+* @param {element} block The block element
+*/
+export function decorateToCalciteBtn(block) {
```

Comment:
See `dom-helpers.js` for existing `calciteButton` function

Reply:
@twhite313 used your script from call-to-action and moved it to script.js rather than have multiple scripts doing essentially the same function.